common/src/test/java/bisq/common/monetary/QuoteTest.java

```diff
@@ -80,7 +80,7 @@ void testFromMarketPriceOffset() {
         Quote quote = Quote.fromMarketPriceOffset(marketQuote, 0);
         assertEquals(500000000, quote.getValue());
         assertEquals(4, quote.getPrecision());
-        assertEquals("BTC/USD", quote.getMarket().toString());
+        assertEquals("US Dollar (BTC/USD)", quote.getMarket().toString());
```

Comment:
I fixed that in the meantime using another mehtod for the currency pair. can you revert those 2 commits?